apps/factory/src/components/prompt_completion.tsx

```diff
@@ -21,15 +23,35 @@ const PromptCompletion: React.FC<PromptCompletionProps> = ({
   if (modelType === ModelTypeSchema.Enum.TEXT2TEXT) {
     return (
       <>
-        {tokens ? (
-          <>
-            {output}
-            <p>tokens: {tokens}</p>
-          </>
+        {window.location.href.includes("/logs") ? (
```

Comment:
@LovedeepSingh19 not sure why this is needed ? please explain

Reply:
@LovedeepSingh19 we should not be usng url based hacks in the components, rather pass a parameter in the components. since we are sure when we are calling it. it doesnot dependent on url in any way.

Reply:
> @LovedeepSingh19 not sure why this is needed ? please explain

without this even in the logs page all the text output will be displayed with chatgpt like animation

Reply:
> @LovedeepSingh19 we should not be usng url based hacks in the components, rather pass a parameter in the components. since we are sure when we are calling it. it doesnot dependent on url in any way.

that issue is only specific to logs page, everywhere else we need animation in prompt_completion component. For example- in SugarCubes the same component is used and there we'd also prefer the animation.
I can definately pass its as a prop if thats needed